ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/cco-manual-workload-identity/tp/cucushift-installer-rehearse-azure-ipi-cco-manual-workload-identity-tp-workflow.yaml

```diff
@@ -0,0 +1,17 @@
+workflow:
+  as:  cucushift-installer-rehearse-azure-ipi-cco-manual-workload-identity-tp
```

Comment:
If I remember right, this file was already merged, why this PR create it again?

Reply:
yes, updating

Reply:
It seems that I used a different repo locally to submit the code to cause this problem.